### PR TITLE
Prep for 22.1 release

### PR DESCRIPTION
**Pull request by Myoldmopar**
Prep for 22.1 release, typical cleanups and readme updates.

### CODE REVIEW

**Comment by Myoldmopar** [comment]
Minimal changes here in preparation for release, merging this and tagging RC1 so we can get some full testing going.

File: src/Transition/InputRulesFiles/Rules9-6-0-to-22-1-0.md [hunk 1]
```diff
@@ -4,7 +4,17 @@ Input Changes version 9.6.0 to 22.1.0
 This file documents the structural changes on the input of EnergyPlus that could affect interfaces, etc.
 This was previously an Excel workbook that made for very difficult version control, especially during busy times around code freezes.
 
-**For WindowProperty:FrameAndDivider:**
+# Object Change: PythonPlugin:SearchPaths
+
+Summary: A new field was added to have EnergyPlus search for Python scripts in the `epin` environment variable value
```
Comment (mjwitte) [inline src/Transition/InputRulesFiles/Rules9-6-0-to-22-1-0.md:9]: This doesn't work on Windows with EP-Launch. Seems like it should?
```
Program Version,EnergyPlus, Version 22.1.0-4ba5fb374c, YMD=2022.03.25 15:27,
   ** Severe  ** Failed to import module "PythonPluginConstantVolumePurchasedAir"
   **   ~~~   ** Python error description follows: 
   **   ~~~   ** ModuleNotFoundError("No module named 'PythonPluginConstantVolumePurchasedAir'")
   **   ~~~   ** In reportPythonError(), traceback.format_exception did not return a list.
   **  Fatal  ** Python import error causes program termination
```

**Comment by Myoldmopar** [comment]
Yeah, I thought that was the point.  I'll have to investigate.

**Comment by mjwitte** [comment]
The schedule:file paths work ok, e.g. 5ZoneTDV runs with EP-Launch now.

File: release/readme.html [hunk 2]
```diff
@@ -380,13 +389,13 @@ <h2><a id="Platforms_90"></a>Platforms</h2>
         not testing on Windows 7 or 8, only 10. However, both our 32-bit and 64-bit packages now include the new
```
Comment (mjwitte) [inline release/readme.html:389]: Windows 11?

**Review by mjwitte — COMMENTED** [review]
@Myoldmopar Some minor edits for the readme for now or next time:

For older conversion programs, there is a full link available now:
https://energyplushelp.freshdesk.com/support/solutions/folders/67000241428

And the testing reports page is rather out of date. Link to the other testing repo directly, or maybe we update this page to add links to the newer reports
https://energyplus.net/testing

"Weather data for more than 2100 locations" --> 3000

The "changelog" link under Known Problems is broken (points to a local changelog file).

File: src/Transition/InputRulesFiles/Rules9-6-0-to-22-1-0.md (hunk 1 above)
Comment (Myoldmopar) [reply src/Transition/InputRulesFiles/Rules9-6-0-to-22-1-0.md:9]: I believe this is fixed in #9349 now, thanks @mjwitte !

File: release/readme.html (hunk 2 above)
Comment (Myoldmopar) [reply release/readme.html:389]: @mjwitte thanks, I've updated the readme with these notes.  I'll put those changes in #9349 ahead of merging it once it's been tested.

**Pull request merged**